src/js/views/eva-variant-view.js

```diff
@@ -36,6 +36,83 @@ EvaVariantView.prototype = {
         return '';
     },
 
+    get_chromosome_number_for_accession: function(chromosome_accession, line_limit) {
+          var ENA_TEXT_API_URL = "https://www.ebi.ac.uk/ena/browser/api/text/" + chromosome_accession + "?lineLimit=" + line_limit + "&annotationOnly=true";
```

Comment:
Did you check that this works for mitochondrial variants? If they have NC_* accessions, this call won't work for them